tmt/steps/provision/mrack.py

```diff
@@ -919,6 +927,34 @@ def create_host_requirement(self, host: CreateJobParameters) -> dict[str, Any]:
 
             # Whiteboard must be added *after* request preparation, to overwrite the default one.
             req['whiteboard'] = host.whiteboard
+            if host.bootc:
+                req['ks_append'] = req['ks_append'] if req.get('ks_append') else []
+                ks_meta = req['ks_meta'] if req.get('ks_meta') else ''
+                req['ks_meta'] = (
+                    'no_default_harness_repo disable_debug_repos disable_onerror ' + ks_meta
+                )
+                ks_list = [
+                    f"""
+ostreecontainer --url {host.image_url}
+%pre
+#!/bin/sh
+cat > /etc/ostree/auth.json <<EOF
+{host.auth_dict}
+EOF
+%end
+"""
+                ]
+                req['ks_append'].extend(ks_list)
+                req['tasks'] = [
+                    {
+                        "name": "/distribution/check-system",
+                        "role": "None",
+                        "fetch_url": "https://gitlab.com/fedora/bootc/tests/bootc-beaker-test/-/archive/1.8/bootc-beaker-test-1.8.tar.gz#check-system",
```

Comment:
Hardcoded version - can it become an option, with `1.8` as a default? Or even the whole location of the `bootc-beaker-test` archive?